service/prometheus/service.go

```diff
@@ -0,0 +1,81 @@
+package prometheus
+
+import (
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+const (
+	prometheusReloadPath = "/-/reload"
+)
+
+type Prometheus interface {
+	Reload() error
+}
+
+type Config struct {
+	Logger micrologger.Logger
+
+	PrometheusAddress string
+}
+
+func DefaultConfig() Config {
+	return Config{
+		Logger: nil,
+
+		PrometheusAddress: "",
+	}
+}
+
+type Service struct {
+	logger micrologger.Logger
+
+	prometheusAddress string
```

Comment:
Same here - `prometheusAddress`, or just `address`?

Reply:
This should mention in the name, that this is now the address + reloadpath IMO

Reply:
Done.